FirebaseSessions/Tests/Unit/SessionsSettingsTests.swift

```diff
@@ -65,8 +65,7 @@ class SessionsSettingsTests: XCTestCase {
   func test_NoRemoteAndDefaultsPresent_DefaultConfigsApply() {
     let emptySettings: [String: Any] = [
       "cache_duration": 10,
-      "app_quality": [
-      ],
```

Comment:
IIUC, the value here was incorrect as it's been changed to a dictionary `[:]` in the new diff?

Reply:
That’s what I believe. `app_quality` is a dictionary in every other test, plus it’s decoded as a dictionary in production code (`RemoteSettings.swift:43`), so I think it was previously declared as an array here simply by mistake.